### DISCUSSION TITLE
Is there a difference between using different seed values for the three drives vs same value?

### DISCUSSION THREAD

**Discussion by mkhalil8** [jonescompneurolab/hnn-core#572] [Q&A]
If I used seed 14 for p1,d1,p2 drives for a simulation then I used different values for each of them, should that make a difference in the dipoles?

**Comment by jasmainak** [comment]
If you use a drive with the same seed values and same parameters, it should have the same input drives ... but if you use different seed values, the drives will be different. Now, whether it makes a difference to your dipole will depend on the specific sequence of drives you have set up for your simulation.

**Reply by mkhalil8** [reply]
So if I have the seed for p1, d1, p2 as 14 for each, then in other simulation make the values as 274, 544, 814, which approach is better? and would that make difference in dipole (more than the normal variation in dipole with change of seeds)?

**Reply by rythorpe** [reply]
They're statistically equivalent. Under the hood, hnn_core reads in the seed value set by the user and set's the state of the random number generator it in a GID-dependent manner. These means that different drives set with the same seed value will end up with different pseudorandom pre-synaptic spike times.

**Reply by jasmainak** [reply]
Your results should not depend on the seed ... take some sort of an average of multiple trials. If you want deterministic drives, set std_dev = 0
